context-free.md

```diff
@@ -1,22 +1,40 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+This programming language seems to be for a general audience, but using its more
+advanced features (the recursion aspects, for example) will require some
+knowledge of CS concepts. The overall grammar seems understandable, though it
+requires a lot of finagling with different parameters.
 
 ## What is easy to do in this language? Why is it easy?
-
+It is easy to generate shapes; you don't need to have a drawing tablet or
+anything in order to create art. You can just define a shape (square, for
+instance) and have it appear on the page.
 
 ## What is hard to do in this language? Why is it hard?
-
+It is hard to create shapes that go beyond the basics. To create more advanced
+shapes, you need to build it using advanced shapes. As stated before, it's a lot
+of playing with different numbers to see what works.
 
 ## How did you learn how to program in this language?
-
+I looked at the provided examples (there were specifically two demo files that
+gave a brief walkthrough on what different parameters do). I also looked at the
+online documentation to see what I could do. 
 
```

Comment:
I learned by looking at the demo files as well. I feel like they were good enough to give me a basic understanding, but there were some other things I wanted to try out like changing the color of certain shapes, but I couldn't figure it out even though it was in the examples.